parser/src/wasm/template.js

```diff
@@ -1,5 +1,8 @@
 /* eslint-disable no-unused-vars,camelcase,no-undef */
 
+const textEncoder = new TextEncoder()
+const textDecoder = new TextDecoder()
+
 function loadWASM() {
   return require('node:fs').readFileSync(require('node:path').resolve(__dirname, 'milo.wasm'))
```

Comment:
For the browser environment this will not work. But we don't need to change this as milo already exports a `index-with-wasm` (probably I should rename this) which embeds the WASM file inside.